ethstorage/archiver/guide-blob-test.md

```diff
@@ -0,0 +1,198 @@
+# Testing EthStorage Archive Service 
+
+## Table of Contents
+
+1. [Introduction](#introduction)  
+2. [Environment Setup](#environment-setup)  
+   2.1. [Running a Proxy for the Ethereum Beacon API](#running-a-proxy-for-the-ethereum-beacon-api)  
+   2.2. [Running an es-node with the Archive Service Enabled](#running-an-es-node-with-the-archive-service-enabled)  
+   2.3. [Set Environment Variables](#set-environment-variables)  
+3. [Testing EthStorage Archive Service](#testing-ethstorage-archive-service)  
+   3.1. [Upload a Blob](#upload-a-blob)  
+   3.2. [Query the Blob Info](#query-the-blob-info)  
+   3.3. [Load the Expired Blob from EthStorage](#load-the-expired-blob-from-ethstorage)  
+   3.4. [Verify the Blob](#verify-the-blob)  
+4. [Conclusion](#conclusion)  
+
+## Introduction
+
+This document outlines the testing strategy and details for the blob archiver service of EthStorage. Its main objective is to verify that the EIP-4844 blobs can be effectively downloaded, stored, and retrieved from EthStorage, even after they have expired and been pruned from the L1 Beacon client.
+
+## Environment Setup
+
+The tests will be conducted in an environment with the following services operational:
+
+- An Ethereum L1 (Sepolia) RPC from an Execution Client running in archive mode.
+- An Ethereum L1 (Sepolia) Beacon API URL.
+- A proxy for the Ethereum Beacon API to simulate a short blob retention period.
+- An EthStorage node (es-node) with the archive service enabled.
+
+The following sections detail the setup of the Beacon proxy and the es-node.
+
+### Running a Proxy for the Ethereum Beacon API
+
+To download and run a proxy for the Beacon client, execute the following commands:
+
+```bash
+git clone https://github.com/ethstorage/beacon-api-wrapper.git
+cd beacon-api-wrapper
+go run cmd/main.go -r 1800
+```
+
+This proxy functions like a standard Beacon API, except that it has a much shorter blob retention period—30 minutes in this case. Consequently, when a `blob_sidecars` request is made for blobs older than 150 slots, it will return an empty list: `{"data":[]}`.
```

Comment:
It's better to make it explicit that 30 min correspond to the `1800` parameter, which is retention period in seconds.